template/.travis.temp

```diff
@@ -0,0 +1,30 @@
+language: node_js
```

Comment:
Fie should be renamed `template/.travis.temp`

Reply:
The script does this when copying

Reply:
I am saying that it's unnecessary. In this new setup there is no reason fo have it as .travis.temp.